ui/src/components/Search/Search.js

```diff
@@ -23,6 +23,9 @@ const Search = withRouter(({ history }) => (
           case 'Responsibility':
             history.push(`/${node.fulfills.nodeId}/${node.nodeId}`);
             break;
+          case 'Person':
+            history.push('/userprofile', { node });
```

Comment:
See my other comments about switching to "/profile/:personId" path and using the :personId url parameter in the UserProfile component. Then you would do this instead: 

```
history.push(`/profile/${node.nodeId}`);
```